packages/mask/shared-ui/locales/en-US.json

```
@@ -51,6 +55,20 @@
     "auto_paste_failed_snackbar": "Do you need to paste encrypted content manually?",
     "auto_paste_failed_snackbar_action": "Show me how",
     "auto_paste_failed_snackbar_action_close": "Close",
+    "compose_encrypt_method_text": "Text",
+    "compose_encrypt_method_text_sub_title": "Use text encryption",
+    "compose_encrypt_method_image": "Image",
+    "compose_encrypt_method_image_sub_title": "Encrypt messages in images",
+    "compose_no_local_key": "No local key",
+    "compose_encrypt_visible_to_all": "All",
+    "compose_encrypt_visible_to_all_sub": "Everyone",
+    "compose_encrypt_visible_to_private": "Private",
+    "compose_encrypt_visible_to_private_sub": "Just Me",
+    "compose_shared_friends_single": "1 freind",
+    "compose_shared_friends_multiple": "{{count}} freinds",
```

Use `Plurals ` feature of `i18next`, and follow the the style of plural suffix.
https://www.i18next.com/translation-function/plurals

```suggestion
    "compose_shared_friends_one": "1 friend",
    "compose_shared_friends_other": "{{count}} friends",
```

And usage will be simpler:

```diff
-return shareWithNum > 1
-                ? t('compose_shared_friends', { count: shareWithNum })
-                : t('compose_shared_friends_single')
+return t('compose_shared_friends',  { count: shareWithNum })
```


You should disable `locale-kit` from commit hook as well. (discussed here https://github.com/DimensionDev/Maskbook/issues/6221)

---

modified